### PR TITLE
Jumpstart

### PR DESCRIPTION
**Pull request by Ulincsys**
**Description**
- Add jumpstart subroutine for more robust process management
- Fix logging inconsistencies
- Fix `augur api start` and `augur collection` start commands
- Add Textual dependency

In preparation for changes coming in the Admin Console release, it is desirable to have a proper "process manager" in place that can communicate with external clients and control Augur processes from a central perspective. This change enables more granular control over Augur's various worker types, and in the future more features are possible.

**Signed commits**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: augur/api/view/init.py [hunk 1]
```diff
@@ -91,4 +91,4 @@ def write_settings(current_settings):
 # Initialize logging
 def init_logging():
     global logger
-    logger = AugurLogger("augur_view", reset_logfiles=True).get_logger()
+    logger = AugurLogger("augur_view", reset_logfiles=False).get_logger()
```
Comment (sgoggins) [inline augur/api/view/init.py:94]: Does this only affect view?

File: augur/application/cli/collection.py [hunk 2]
```diff
@@ -22,14 +22,15 @@
 from augur.application.db.session import DatabaseSession
 from augur.application.logs import AugurLogger
 from augur.application.db.lib import get_value
-from augur.application.cli import test_connection, test_db_connection, with_database
+from augur.application.cli import test_connection, test_db_connection, with_database, DatabaseContext
```
Comment (sgoggins) [inline augur/application/cli/collection.py:25]: Is this one just incorporating the latest from @ABrain7710 ?

**Review by sgoggins — DISMISSED** [review]
I think @ABrain7710 needs to check the backend.py edits.

File: augur/application/cli/collection.py (hunk 2 above)
Comment (Ulincsys) [reply augur/application/cli/collection.py:25]: No, the changes are unrelated

File: augur/api/view/init.py (hunk 1 above)
Comment (Ulincsys) [reply augur/api/view/init.py:94]: No, it previously would reset all log files globally. So now the log files will continue to exist after starting Augur

**Pull request merged**